neo/Network/P2P/Payloads/Block.cs

```diff
@@ -11,10 +11,12 @@ namespace Neo.Network.P2P.Payloads
 {
     public class Block : BlockBase, IInventory, IEquatable<Block>
     {
-        public const int MaxTransactionsPerBlock = ushort.MaxValue;
+        public const int MaxContentsPerBlock = ushort.MaxValue;
+        public const int MaxTransactionsPerBlock = MaxContentsPerBlock - 1;
 
-        public ConsensusData ConsensusData;
-        public Transaction[] Transactions;
+        public IBlockContent[] Contents;
```

Comment:
What data is expected to save in this array?, i think that is better to store these things in separate properties